gossip/src/crds.rs

```diff
@@ -303,6 +309,25 @@ impl Crds {
                     Err(CrdsError::InsertFailed)
                 } else if matches!(route, GossipRoute::PushMessage(_)) {
                     let entry = entry.get_mut();
+                    if entry.num_push_dups == u8::MAX {
+                        datapoint_info!(
+                            "gossip_crds_redundant_pull",
+                            (
+                                "origin",
+                                value.value.pubkey().to_string().get(..8),
+                                Option<String>
+                            ),
+                            (
+                                "signature",
+                                value.value.signature.to_string().get(..8),
+                                Option<String>
+                            )
```

Comment:
these effectively random strings do not compress well in the metrics db and are difficult to query. i'd highly recommend rethinking the accounting to get the information you need without them

Reply:
did not think about this. good to know. Will try to do something like what behzad mentioned: https://github.com/anza-xyz/agave/pull/139#discussion_r1517981177